### ISSUE TITLE
[Mac] 鼠须管新版本Caps点亮输出小写，怎么破？

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lotem/rimeime#343]
```
0.9.15 “新增：Caps Lock 
點亮時，切換到西文模式，輸出小寫字母”

看了这一行what's new，不敢更新了。
我经常直接用Caps 
Lock来输入大写字母，但难道这个新版本会默认变小写？我简�
��翻了一下wiki，没有相关的配置说明。
因为我肯定接受不了Caps还出小写，所以先坐等配置方法，再�
��新版本，

#請提供以下信息，幫我們定位問題。

1. 輸入法的款式：鼠鬚管

2. 操作系統類型：Mac
   操作系統版本：10.8.2

3. 輸入方案名稱：朙月拼音 简化字


```

Original issue reported on code.google.com by `ltfpe...@gmail.com` on 16 Jan 2013 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
這次的修改意圖在與Apple的輸入法保持一致。
只有選擇了中文輸入法才會改變Caps 
Lock的作用。不影響其他語言和鍵盤。

如果實在在意這個，就先別更新了。
```

Original comment by `chen....@gmail.com` on 16 Jan 2013 at 8:10

**Comment by GoogleCodeExporter** [comment]
```
沒仔細讀更新說明，caps 
lock失效了之後才發現是因爲rime，因爲特意切換到美國英文就
ok

ls的說法當然不錯，但是有無考慮到　很多用戶關閉了所有其
它的輸入法，只留rime，中英文輸入可以通過修改詞典*.schema.y
aml的reset從0變1來設定默認英文：

switches:
  - name: ascii_mode
    reset: 1
    states: [ 中文, 西文 ]

更新之前只有rime輸入法，默認英文，需要的時候調用中文；
更新之後只有rime輸入法，默認英文，大寫琐定失效。

希望可以在下個版本修正，多謝。

1. 輸入法的款式：鼠鬚管

2. 操作系統類型：Mac
   操作系統版本：10.8.2
```

Original comment by `cccddd1...@gmail.com` on 16 Jan 2013 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
可以添加一個選項。我想知道系統自帶輸入法怎樣處理這個��
�題的。
```

Original comment by `chen....@gmail.com` on 16 Jan 2013 at 9:54

**Comment by GoogleCodeExporter** [comment]
```
查了下，有 Shift+CapsLock 一说。看似靠谱。
```

Original comment by `chen....@gmail.com` on 16 Jan 2013 at 10:17

**Comment by GoogleCodeExporter** [comment]
```
建议还是至少搞一个可以custom的开关吧，比较保险。辛苦了��
�
```

Original comment by `ltfpe...@gmail.com` on 16 Jan 2013 at 10:21

**Comment by GoogleCodeExporter** [comment]
Original comment by `chen....@gmail.com` on 16 Jan 2013 at 2:29
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
fixed in Squirrel 0.9.15.1, Weasel 0.9.19.1

disable mode switching (thus preserving UPPERCASE):
{{{
# default.custom.yaml
patch:
  ascii_composer/switch_key/Caps_Lock: noop
}}}

or switch to "ASCII" mode:

{{{
#default.custom.yaml
patch:
  ascii_composer/good_old_caps_lock: true
}}}

```

Original comment by `chen....@gmail.com` on 17 Jan 2013 at 2:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
👍👍👍👍👍
```

Original comment by `ltfpe...@gmail.com` on 17 Jan 2013 at 2:08
